pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -380,10 +387,11 @@ func (vm *windows) createService(svc *service) error {
 	if svc == nil {
 		return errors.New("service object should not be nil")
 	}
-	out, err := vm.Run("sc.exe create "+svc.name+" binPath=\""+svc.binaryPath+" "+
-		svc.args+" start=auto", false)
+	svcCreateCmd := "sc.exe create " + svc.name + " binPath=\"" + svc.binaryPath + " " + svc.args + " start=auto"
+	out, err := vm.Run(svcCreateCmd, false)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create service with output: %s", out)
+		log.Info("failed to create service", "cmd", svcCreateCmd, out)
```

Comment:
odd number of log Info pairs.

Reply:
Looking at this log again, I should remove it. The error will be logged by `vm.Run()`.